presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -394,7 +402,15 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .functions(ROW_HASH_CODE, ROW_TO_JSON, ROW_EQUAL, ROW_NOT_EQUAL)
                 .function(CONCAT)
                 .function(DECIMAL_TO_DECIMAL_CAST)
-                .function(TRY_CAST);
+                .function(TRY_CAST)
+                .function(DECIMAL_AVERAGE_AGGREGATION)
+                .function(DECIMAL_SUM_AGGREGATION)
+                .functions(DECIMAL_CEILING_FUNCTIONS)
+                .function(DECIMAL_FLOOR_FUNCTION)
+                .function(DECIMAL_ABS_FUNCTION)
+                .functions(DECIMAL_POWER_FUNCTIONS)
+                .function(DECIMAL_MOD_FUNCTION)
+                .functions(DECIMAL_ROUND_FUNCTIONS);
```

Comment:
Maybe just put all the functions in a single `functions` call. It will be less verbose.